schema/workflow/unit/io/object_storage.json

```diff
@@ -0,0 +1,33 @@
+{
+    "schemaId": "workflow-unit-io-object-storage",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "object_storage io schema",
+    "type": "object",
+    "allof": [
+        {
+            "$ref": "../../unit.json"
+        }
+    ],
+    "properties": {
+        "pathname": {
+            "description": "Name of the directory to contain the file when it is downloaded. If empty, the current directory is used",
+            "type": "string"
+        },
+        "basename": {
+            "description": "Basename the file should have when it is downloaded",
+            "type": "string"
+        },
```

Comment:
Let's isolate pathname/basename to its own shema for reusability

Reply:
Isolated to its own schema